src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java

```diff
@@ -29,10 +29,10 @@ public void createNewWikiLangTC001(String lang) {
     if (lang.equals("en")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "en" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)